docs/source/tutorials_wine.rst

```diff
@@ -0,0 +1,590 @@
+with Jupyter Lab plugin.
+
+Prepare code and data
+######################
+
+Prepare MLFlow project
+------------------------
+
+In this tutorial we will solve the famous `hello world` problem of wine classification using Legion Platform with MLFlow integration.
+
+First of all let's review project source code.
+To to this you should clone legion-examples github repository. This repo includes examples of projects that are ready to be used
+with Legion Platform. We will focus on wine example that is located at ``mlflow/sklearn/wine/`` directory inside the repo.
+So let's also change our directory to that after repo cloning and look around with ``tree`` command
+
+.. code-block:: console
+
+   $ git clone https://github.com/legion-platform/legion-examples && cd legion-examples
+   $ cd mlflow/sklearn/wine/
+   $ tree
+
+We will se next structure of project
+
+.. code-block:: text
+    :caption: ../mlflow
+    :name: MLproject example file
+
+    .
+    ├── MLproject
+    ├── airflow
+    │   └── dag.py
+    ├── conda.yaml
+    ├── data
+    │   └── wine-quality.csv
+    ├── legion
+    │   ├── deployment.legion.yaml
+    │   ├── packaging.legion.yaml
+    │   ├── training.legion.yaml
+    │   └── wine_connection.legion.yaml
+    ├── test
+    │   └── test_wine.py
+    ├── train.py
+    └── wine-quality.csv
+
+In this directory, actually only some files are interested for us in the context of the tutorial:
+
+- Three files that belongs to MLFlow project itself (see MLFlow docs for extra info)
+    - MLproject
+    - conda.yaml
+    - train.py
+- Legion yaml files that we will use as a payload for Legion API in training, packaging and deployment stages
+    - legion/training.legion.yaml
+    - legion/packaging.legion.yaml
+    - legion/deployment.legion.yaml
+    – legion/wine_connection.legion.yaml
+    – legion/vcs_connection.legion.yaml
+    – legion/docker_connection.legion.yaml
+– File where input data for learning is saved
+    – data/wine-quality.csv
+
+To learn more about MLFlow project structure you can use next official MLFlow documentation.
+
+We'll show this files with a bit comments.
+
+.. code-block:: console
+
+   $ cat MLproject
+
+.. code-block:: yaml
+    :caption: MLproject
+    :name: MLproject example file
+
+    name: wine-quality-example
+    conda_env: conda.yaml
+    entry_points:
+        main:
+            parameters:
+                alpha: float
+                l1_ratio: {type: float, default: 0.1}
+            command: "python train.py --alpha {alpha} --l1-ratio {l1_ratio}"
+
+
+.. code-block:: console
+
+   $ cat conda.yaml
+
+.. code-block:: yaml
+   :caption: ../conda.yaml
+   :name: Conda environment for current project
+
+    name: example
+    channels:
+    - defaults
+    dependencies:
+    - python=3.6
+    - numpy=1.14.3
+    - pandas=0.22.0
+    - scikit-learn=0.19.1
+    - pip:
+        - mlflow==1.0.0
```

Comment:
Please note that all packages (system and python) that user wanna use should be described in this conda env file.
Also, please provide reference to off. docs